setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
i prefer iris-scitools

the convention is to just name the package the same as the import, but we can't do that, due to a clash.

I'd prefer to have `iris` first in the name